pkg/gccbuilder/gcc-builder.go

```diff
@@ -333,7 +341,7 @@ func (g *GccBuilder) parseCommandLine(args []string) {
 			}
 		case Compile:
 			for _, input := range g.Input {
-				objectname := strings.TrimSuffix(input, filepath.Ext(input)) + ".s"
+				objectname := strings.TrimSuffix(input, filepath.Ext(input)) + ".o"
```

Comment:
Why did you change the "s" to "o"? 
In the compilation step the output file has suffix "s" and then in the assemble step this file is converted to an object file. Or I got it wrong?
From the documentation: http://codingfreak.blogspot.com/2008/02/compilation-process-in-gcc.html
By using “-S” flag with gcc we can convert the preprocessed C source code into assembly language without creating an object file:
$ gcc -Wall -S hello.i -o hello.s

Reply:
Oops. I don't know how this happened. Good catch.